### ISSUE TITLE
Unraid docker install, cannot create icloudpd.conf on initialze

### ISSUE THREAD

**Issue by tex247** [boredazfcuk/docker-icloudpd#596]
Installed unraid docker 
Getting this error when running sync-icloud.sh --Initialise

2024-07-01 11:27:57 INFO     ***** boredazfcuk/icloudpd container v1.0.876_01-07-2024 started *****
2024-07-01 11:27:57 INFO     ***** For support, please go here: https://github.com/boredazfcuk/docker-icloudpd *****
2024-07-01 11:27:57 INFO     Alpine Linux 3.20.1
2024-07-01 11:27:57 INFO     Python version: 3.12.3
2024-07-01 11:27:58 INFO     icloud-photos-downloader version: 1.20.4
2024-07-01 11:27:58 ERROR    Failed to create configuration file: /config/icloudpd.conf - Cannot continue, exiting.

I changed /config folder permissions to 777 so not sure why script cannot create file. Advice? Thank you!

**Comment by tex247** [comment]
folder permission currently looks like this:
drwxrwxrwx   1   99 users   0 Jul  1 11:27 config

**Comment by DominicWoozy** [comment]
same here

**Comment by DominicWoozy** [comment]
i figured it out, you can comment out 
```bash
if ! test -w "${config_file}"; then
    echo "Cannot write to ${config_file} - Please correct your permissions"
    sleep 600
    exit 1
fi
```
in /usr/local/bin/init_config.sh directly, and run this bash file again to generate config file

**Comment by boredazfcuk** [comment]
Yeah, it didn't work as intended when this code was in the init_config.sh script. It didn't exit the container, just the script, so it then launches the main script without creating the config file. I then moved the code to the main script, and changed it so that it just checks whether the config file has been created or not. Left the above code in by mistake. Should be fixed in latest version I just pushed to Dockerhub.
